internal/cli/command/execution/execution.go

```diff
@@ -40,4 +51,10 @@ func NewCmd(octaviusDaemon daemon.Client) *cobra.Command {
 			printer.Println(fmt.Sprintf("Job has been added to pending list successfully.\nYou can see the execution logs using getstream %s", response.Status), color.FgGreen)
 		},
 	}
+
+	executionCmd.Flags().StringVarP(&jobName, "job-name", "", "", "It contains Job Name")
+	executionCmd.MarkFlagRequired("job-name")
```

Comment:
handle the errors returned. It will be good in making a robust application

Reply:
this is handled in #87